src/index.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import rootReducer from './reducers/index';
 import './index.css';
-import App from './App';
+import App from './components/App';
+
+const store = createStore(rootReducer);
 
```

Comment:
Let's initialize our state in here;
![image](https://user-images.githubusercontent.com/31897434/79816470-3dd7f280-837b-11ea-9e1c-f670756fbe4b.png)